### DISCUSSION TITLE
unable to setup free organization

### DISCUSSION THREAD

**Discussion by Alex4DevOps** [community/community#54597] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have spent whole day trying to set up free organization as long as I select the free organization set up even I have completed all required filed correctly and check the terms and condition check box the Next button never get activated, if I select the pay option the Next button activated right away. my question is since you're not allowing to set up free organization with limited features why keep that un-setup able free option, it is very disrespectful
![Screenshot 2023-05-05 220756-1](https://user-images.githubusercontent.com/131609021/236593405-e35021e9-86f9-4ba0-994b-f33e0f37e1f6.png)

**Comment by Alex4DevOps** [comment]
FYI: I have verified my account twice before trying to set up the organization

**Comment by ryanlua** [comment, marked as answer]
It seems like your captcha isn't loading so that is why the button is dimmed, there is supposed to be a captcha in the grey box.

Try disabling all extensions like ad blockers. If that doesn't work, try creating one in an incognito window or different browser.

**Reply by Alex4DevOps** [reply]
thanks for the reply I will try that

**Comment by Alex4DevOps** [comment]
Thanks Ryan,

I have switched my browser from chrome to MS-Edge and the captcha loaded
correctly, and also solved my other problems with pushing files to github,
there are some problems with my google chrome browser.

*Best regards*
*Alex A. Boghda*

**Reply by ryanlua** [reply]
No problem, if you feel like that my answer was satisfactory you can select "mark as answer" on my answer to pin it and mark this discussion as answered .

**Comment by Alex4DevOps** [comment]
Will do, thanks!

On Sat, May 6, 2023 at 3:18 AM Ryan ***@***.***> wrote:

> No problem, if you feel like that my answer was satisfactory you can
> select "mark as answer" on my answer to pin it and mark this discussion as
> answered .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/54597#discussioncomment-5822678>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7MDDPLFDYL4IHTXHLRIDATXEX3LPANCNFSM6AAAAAAXXZCRVQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Best regards*
*Alex A. Boghda*

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Alex4DevOps! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
